### DISCUSSION TITLE
Multi Sensor Tracking

### DISCUSSION THREAD

**Discussion by Jadams29** [mikel-brostrom/yolo_tracking#552] [Q&A]
I have a trained YOLO model that I am loading. My issue is keeping the same IDs for detections over various frames.

I have combined my 4 sensors data into a single video. This allows me to run StrongSORT on a singular image rather than 4 separate. I tried running on 4 separate and each time a frame was processed no detections from StrongSORT were made. I get that this could be part of the many parameters StrongSORT such as max distance or max IOU distance. I have tried many variations of parameters and they do not seem to make much, if any difference. 

My next idea was to process all four frames and in some fashion combine their feature vectors. Such that all features vectors for vehicle A would be some combination of the vectors from each frame. Then keep a running combination of previous feature vectors and new ones until the age is met. 

I would appreciate any suggestions to resolve this issue.

ok my video is not loading...
https://vimeo.com/755981733

**Comment by tgrmev** [comment]
@Jadams29 I watched the video attached to your question and it is similar what I want to do. Could you explain, how did you track the objects for multiple camera by using this repository?

**Comment by mikel-brostrom** [comment]
https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/issues/594#issuecomment-1310624465
